fs/config.go

```diff
@@ -635,6 +635,7 @@ type ConfigInfo struct {
 	Inplace                    bool              `config:"inplace"`      // Download directly to destination file instead of atomic download to temp/rename
 	PartialSuffix              string            `config:"partial_suffix"`
 	MetadataMapper             SpaceSepList      `config:"metadata_mapper"`
+	UnixSocketPath             string            `config:"unix_socket_path"` // Optional unix socket to connect to
```

Comment:
I don't think you want this global config so remove this

Reply:
Done